Adds Ember Data Medium Term Dev Plan RFC 452 (#17)
ember-learn/ember-blog: source/2019-03-15-the-ember-times-issue-89.md

---

> ## [Ember Data Medium Term Development Plan RFC](https://github.com/emberjs/rfcs/pull/452)⚡️💾

Reviewer: ```suggestion
## [Ember Data Medium Term Development Plan RFC ⚡️💾](https://github.com/emberjs/rfcs/pull/452)
```

> [@igorT](https://github.com/igorT) recently opened an RFC setting a medium term plan for Ember Data's development with the goals of increasing approachability, speed, stability flexibility and shedding some legacy layers.

Reviewer: ```suggestion
[@igorT](https://github.com/igorT) recently opened an RFC setting a medium term plan for Ember Data's development with the goals of increasing approachability, speed, stability, flexibility and shedding some legacy layers.
```

> The RFC describes a tactical plan to refactor Ember Data's internals to isolate DS.Model to facilitate swapping it out in the future, replacing InternalModel functionality, introducing RecordIdentity to uniquely identify records and harden system boundaries.

Reviewer: ```suggestion
The RFC describes a tactical plan to refactor Ember Data's internals to isolate `DS.Model` to facilitate swapping it out in the future, replace`InternalModel` functionality, and introduce `RecordIdentity` to uniquely identify records and harden system boundaries.
```

> For more details and some interesting diagrams sketching out the architecture of what's involved, check out the [RFC on Ember Data's medium term development plan.](https://github.com/emberjs/rfcs/pull/452)

Reviewer: ```suggestion
For more details and some interesting diagrams sketching out the architecture of what's involved, check out the [RFC on Ember Data's medium term development plan](https://github.com/emberjs/rfcs/pull/452).
```

> For more details and some interesting diagrams sketching out the architecture of what's involved, check out the [RFC on Ember Data's medium term development plan](https://github.com/emberjs/rfcs/pull/452).

Reviewer: ```suggestion
For more details and some **interesting diagrams** sketching out the architecture of what's involved, check out the [RFC on Ember Data's medium term development plan](https://github.com/emberjs/rfcs/pull/452).
```
